pegasus/sites.v3/hourofcode.com/views/home_highlights.haml

```diff
@@ -2,8 +2,12 @@
   .row
     .col-md-8
       %br/
-      %h2 Hour of Code 2017
-      %a.twitter-grid{href: "https://twitter.com/codeorg/timelines/925449654308704256?limit=4?ref_src=twsrc%5Etfw?limit=4"} The Hour of Code 2017
+      %h2
```

Comment:
Oh, actually thinking about it more this section is about last year's Hour of Code highlights...